core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java

```diff
@@ -82,7 +83,7 @@ public NativeImageBuildItem build(NativeConfig nativeConfig, NativeImageSourceJa
             // E.g. "/usr/bin/docker run -v {{PROJECT_DIR}}:/project --rm quarkus/graalvm-native-image"
             nativeImage = new ArrayList<>();
             Collections.addAll(nativeImage, containerRuntime, "run", "-v",
-                    outputDir.toAbsolutePath() + ":/project:z");
+                    FileUtil.translateToUnixStyle(outputDir.toAbsolutePath().toString()) + ":/project:z");
```

Comment:
I would be more comfortable if we only did that for Windows, considering we already have platform specific tests below.

Reply:
+1 - doing so we can remove this condition https://github.com/quarkusio/quarkus/pull/5458/files#diff-fa19c024de216cdbcfde070394b65459R71